instagram/src/components/PostContainer/Posts.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import Post from './Post';
+
+const Posts = props => {
+    return(
+        <div className='posts'>
+            {props.posts.map((p, i) => (
+                <Post post={p} key={i}/>
+            ))}
+        </div>
+    )
+}
+
+Posts.propTypes = {
```

Comment:
Good job getting your proptypes working. Looks like this is a master prop check, you're matching the data exactly!